microsoft/testsuites/core/storage.py

```diff
@@ -345,6 +359,119 @@ def hot_add_disk_parallel_premium_ssd(self, log: Logger, node: Node) -> None:
             log, node, DiskType.PremiumSSDLRS, self.DEFAULT_DISK_SIZE_IN_GB
         )
 
+    @TestCaseMetadata(
+        description="""
+        This test case will verify mount azure nfs 4.1 on guest successfully.
+        """,
+        timeout=TIME_OUT,
+        requirement=simple_requirement(disk=DiskStandardSSDLRS()),
+        priority=2,
+    )
+    def verify_azure_file_share_nfs(self, log: Logger, result: TestResult) -> None:
+        environment = result.environment
+        assert environment, "fail to get environment from testresult"
+        assert isinstance(environment.platform, AzurePlatform)
+        platform = environment.platform
+        information = environment.get_information()
+        resource_group_name = information["resource_group_name"]
+        location = information["location"]
+        random_str = "".join(
+            random.choices(string.ascii_lowercase + string.digits, k=10)
+        )
+        storage_account_name = f"lisasc{random_str}"
+        file_share_name = f"lisa{random_str}fs"
+        log.debug(f"creating storage account {storage_account_name}")
+        # create storage account and file share
+        check_or_create_storage_account(
```

Comment:
It should be wrapped in a feature called like Nfs, instead of call Azure method in the test case. Although it's not related to vm size, but it can be attached to node as a feature.

Reply:
add Nfs feature